### PR TITLE
update module manifest for v11 structures and pf2e v5+ ownership

### LINKED ISSUE #1: ver 11
odule "pf2e-magaambya" uses pack.private which has been replaced with pack.ownership

### PR DESCRIPTION
**Pull request by baronfel**
Closes #1 

Hello! This PR should update this module to Foundry v11 stable (11.306) as well as the pf2e system version 5.2.3. Both of these are the minimums for their respective pairing (e.g. 5.2.3 is the first available PF2e system for foundry 11, and it requires 11.306). This should hopefully result in the broadest compatibility.

All of the _content_ of the module remains the same - I have done three things mostly:

* updated the `private` field to the new permissions structures, following along with [the docs]() at the PF2E system wiki
* put the compendiums contributed by this module into a folder for the module for easier location:
image
* updated the changelogs to describe the changes

With that done I was able to make a release on [my fork](https://github.com/baronfel/pf2e-magaambya/releases/tag/1.6.0) and then test the manifest + contents on my local Foundry instance. All seems well!

image

I was able to use the macro to award an actor with the two schools, and use the feats as I am in my 10.x foundry game.

The maps import and are usable: 
image

The character journals still import and seem to have the same content they do in 10.x:
image

Please let me know what other things I should test

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -7,6 +7,10 @@ on:
 jobs:
   build:
     runs-on: ubuntu-latest
+    permissions:
+        contents: write
+        actions: write
```
Comment (baronfel) [inline .github/workflows/main.yml:12]: The GITHUB_TOKEN needed permissions to update the release - I believe this should do it.

File: module.json [hunk 2]
```diff
@@ -5,25 +5,36 @@
...
+			"name": "PF2e Magaambya",
+			"sorting": "m",
+			"packs": [
+				"Maps",
+				"journals",
+				"Macros",
+				"Feats"
+			]
+		}
+	],
```
Comment (baronfel) [inline module.json:37]: Here's the folder definition that results in the image from the description.

File: module.json [hunk 3]
```diff
@@ -32,37 +43,46 @@
 			"system": "pf2e",
 			"type": "Adventure",
 			"private": true,
-			"flags": {}
+			"ownership": {
+				"PLAYER": "LIMITED",
+				"ASSISTANT": "OWNER"
+			}
```
Comment (baronfel) [inline module.json:49]: These ownership rules were patterned after [the pf2e wiki docs](https://github.com/foundryvtt/pf2e/wiki/Creating-a-PF2e-Content-Module#step-1-preparing-the-module), but I think they should be correct. 

Players should be able to use journals/macros/etc, but only see the feats.

File: module.json [hunk 4]
```diff
@@ -32,37 +43,46 @@
...
-			"private": false,
-			"flags": {}
+			"ownership": {
+				"PLAYER": "OBSERVER",
+				"ASSISTANT": "OWNER"
+			}
 		}
 	],
-	"url": "https://github.com/MrVauxs/pf2e-magaambya",
+	"url": "https://github.com/baronfel/pf2e-magaambya",
```
Comment (baronfel) [inline module.json]: need to revert this, one sec.
```suggestion
	"url": "https://github.com/MrVauxs/pf2e-magaambya",

```

**Comment by baronfel** [comment]
Oh lol - conflicts show that you've done this already! Awesome :)

**Comment by MrVauxs** [comment]
Yeah, sorry for wasting your time but when you made the accidental PR I went ahead and gone into GitHub and updated the module.

The minimum version requirements will stay the same until I open up the module again and actually transition to V11. So far, there is no need for an upgrade. 

Still, appreciate the effort and very nice PR!

**Comment by baronfel** [comment]
No worries at all! This was actually my first time doing any Foundry module development so there was a lot of discovery and reading along the way. This was a good bite-size way to learn the basics.

Thanks for this module, by the way - it's made it a lot easier for my group to track progress in our game:)

**Pull request closed without merging**